README.md

```diff
@@ -205,8 +205,12 @@ postgres=# select * from t;
 ## Running tests
 
 ```sh
+# install poetry if not available on the system.
+# pip install poetry
+
 git clone --recursive https://github.com/neondatabase/neon.git
 make # builds also postgres and installs it to ./tmp_install
+poetry install # install the test dependencies
```

Comment:
It shouldnt work without `--no-root` unless newer poetry versions deal with that automatically. There is shortcut script for that: [scripts/pysync](https://github.com/neondatabase/neon/blob/main/scripts/pysync)

Reply:
Maybe worth linking this paragraph from earlier part of the readme https://github.com/neondatabase/neon#dependency-installation-notes

Reply:
aha, thanks for pointing it out. 
Somehow I missed the dependency installation notes. I will link it in the PR.